test/integration/evaluators/RecursiveEvaluator.test.ts

```diff
@@ -21,6 +21,7 @@ describe('recursive evaluators', () => {
         cache: new LRUCache(),
         discoverer: _ => 'term',
       },
+      defaultTimeZone: { zoneMinutes: 0, zoneHours: 0 },
```

Comment:
This comes back a lot. Make this a const.

Reply:
Or make it an optional argument in the evaluator, and use this by default.

Reply:
Seems like the `SyncRecursiveEvaluator` expects a complete context though :'(